libiocage/lib/JailConfigJSON.py

```diff
@@ -24,7 +24,9 @@ def read(self):
 
     def read_data(self):
         with open(JailConfigJSON.__get_config_json_path(self), "r") as conf:
-            return json.load(conf)
+            data = json.load(conf)
+            conf.close()
```

Comment:
The `with` statement already closes once you leave it, that shouldn't be necessary.